materialsLibrary/src/custom/pbrCustomMaterial.ts

```diff
@@ -109,7 +109,16 @@ export class PBRCustomMaterial extends PBRMaterial {
         return arr;
     }
 
-    public Builder(shaderName: string, uniforms: string[], uniformBuffers: string[], samplers: string[], defines: MaterialDefines | string[], attributes?: string[]): string {
+    public Builder(shaderName: string, uniforms: string[], uniformBuffers: string[], samplers: string[], defines: MaterialDefines | string[], attributes?: string[], options?: BABYLON.ICustomShaderNameResolveOptions): string {
+        options!.processFinalCode = (type: string, code: string) => {
```

Comment:
This will break if no options are passed in. You should check for null before assigning here ?